services/worker/src/worker/job_runners/dataset/config_names.py

```diff
@@ -4,7 +4,7 @@
 import logging
 from typing import Optional
 
-from datasets import get_dataset_config_names
+from datasets import get_dataset_config_names, get_dataset_default_config_name
```

Comment:
it's not clear from https://github.com/huggingface/datasets/blob/f45bc6caa25115a04c41b278671a5a89457eb66c/src/datasets/inspect.py#L371 when the result i None. 

I was wondering when we should reorder, and when not to do it.

I imagine that we should do it only when `default: true` is present in the YAML file, no? Is it the case?

Reply:
It's None when there are multiple configs and no default config.

A default config is a config named "default" or a config with `default: true`

Reply:
OK thanks! Opened: https://github.com/huggingface/datasets/pull/6722